packages/main/src/MultiComboBoxTemplate.tsx

```diff
@@ -0,0 +1,118 @@
+import decline from "@ui5/webcomponents-icons/dist/decline.js";
+import MultiComboboxPopoverTemplate from "./MultiComboBoxPopoverTemplate.js";
+import type MultiComboBox from "./MultiComboBox.js";
+import Tokenizer from "./Tokenizer.js";
+import Token from "./Token.js";
+import Icon from "./Icon.js";
+
+export default function MultiComboBoxTemplate(this: MultiComboBox) {
+	return (<>
+		<div class="ui5-multi-combobox-root ui5-input-focusable-element">
+			<span id="ui5-multi-combobox-hiddenText-nMore" class="ui5-hidden-text">{this._tokensCountText}</span>
+
+			{this.hasValueState &&
+				<span id="ui5-multi-combobox-valueStateDesc" class="ui5-hidden-text">{this.ariaValueStateHiddenText}</span>
+			}
+
+			<Tokenizer
+				slot="_beginContent"
+				preventPopoverOpen={true}
+				hidePopoverArrow={true}
+				readonly={this.readonly}
+				class="ui5-multi-combobox-tokenizer"
+				disabled={this.disabled}
+				onShowMoreItemsPress={this._showFilteredItems}
+				opener={this.morePopoverOpener}
+				popoverMinWidth={this._inputWidth}
+				onTokenDelete={this._tokenDelete}
+				onBeforeMorePopoverOpen={this.handleBeforeTokenizerPopoverOpen}
+				onFocusOut={this._tokenizerFocusOut}
+				onFocusIn={this._tokenizerFocusIn}
+				onClick={this._click}
+				onPaste={this._handlePaste}
+				onKeyDown={this._onTokenizerKeydown}
+				expanded={this._tokenizerExpanded}
+				open={this.tokenizerOpen}
+			>
+				{this.items.map((item, index) => {
+					if (item.isGroupItem) {
+						return item.items?.map((groupItem, groupItemIndex) => {
+							return groupItem.selected && <Token
+								readonly={this.readonly}
+								class="ui5-multi-combobox-token"
+								data-ui5-id={groupItem._id}
+								part={`token-${groupItemIndex}`}
+								text={groupItem.text}
+							/>;
+						});
+					}
+
+					return item.selected && <Token
+						readonly={this.readonly}
+						class="ui5-multi-combobox-token"
+						data-ui5-id={item._id}
+						part={`token-${index}`}
+						text={item.text}
+					/>;
+				})}
+			</Tokenizer>
+
+			<input
+				id="ui5-multi-combobox-input"
+				value={this.value}
+				inner-input
+				placeholder={this._getPlaceholder}
+				disabled={this.disabled}
+				readonly={this.readonly}
+				value-state={this.valueState}
```

Comment:
The `value-state` attribute on the input element looks like some left over when the Multi used to compose Input web component - @MapTo0 do we need this?